requirements.txt

```diff
@@ -14,3 +14,4 @@ scikit-fuzzy
 sklearn
 sympy
 tensorflow
+ordered_set
```

Comment:
Looks like it breaks the alphabetical ordering on requirements.txt.

Why do we need to add this module?